config/provisioning.yaml

```diff
@@ -0,0 +1,72 @@
+rancher:
+  host: ""
+  adminToken: ""
+  insecure: true
+  cleanup: true
+
+#######################
```

Comment:
We can probably just remove these comment blocks tbh. It's just taking up extra lines in this templated YAML. This applies to all of the YAML files, just leaving a single comment here instead of multiple ones.